seqr/views/apis/staff_api_tests.py

```diff
@@ -380,3 +640,31 @@ def test_success_story(self):
 
         self.assertEqual(len(response_json['rows']), 1)
         self.assertDictEqual(response_json['rows'][0], EXPECTED_SUCCESS_STORY)
+
+    @mock.patch('seqr.views.apis.staff_api.export_multiple_files')
+    @responses.activate
+    def test_anvil_export(self, mock_export_multiple_files):
+        url = reverse(anvil_export, args=[PROJECT_GUID])
+        _check_login(self, url)
+
+        # Create a SavedVariant row needed by this test
+        saved_variant = SavedVariant.objects.get(guid__exact = "SV0000001_2103343353_r0390_100")
+        variant_tag_type = VariantTagType.objects.get(name__exact = "Known gene for phenotype")
+        variant_tag = VariantTag.objects.create(variant_tag_type = variant_tag_type)
+        variant_tag.saved_variants.add(saved_variant)
+        variant_tag.save()
+
+        # We will test the inputs of the export_multiple_files method.
+        # Outputs of the method are not important for this test.
+        mock_export_multiple_files.return_value = HttpResponse("Dummy text", content_type="text/plain")
```

Comment:
If output doesn't matter then you shouldn't need to specify any return value at all

Reply:
The API still needs an HttpResponse even the test doesn't need any return value. I will update the comments to specify that.